### PR TITLE
Enable ESLint & TS for smoke tests

### PR DESCRIPTION
**Pull request by develohpanda**
What it says on the tin, minimal changes

### CODE REVIEW

File: packages/insomnia-smoke-test/tsconfig.build.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "dist",
+    "rootDir": ".",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true, // this is required because spectron depends on electron but it is not locatable by typescript for the purpose of types
+    "noImplicitAny": false
```
Comment (develohpanda) [inline packages/insomnia-smoke-test/tsconfig.build.json:10]: ‼️ temporary

Comment (dimitropoulos) [reply packages/insomnia-smoke-test/tsconfig.build.json:10]: cool

**Review by dimitropoulos — APPROVED** [review]
the only thing I found/could think of was: should the package.json have `main` and `types` entries?  non-blocking.

**Comment by develohpanda** [comment]
> the only thing I found/could think of was: should the package.json have `main` and `types` entries? non-blocking.

Not needed strictly because this doesn't actually output anything. The entire package is a package of test suites.

**Pull request merged**